crates/ursa-gateway/rustfmt.toml

```diff
@@ -0,0 +1,8 @@
+condense_wildcard_suffixes = true
```

Comment:
is this just formatting for gateway. Should we maintain that across all crates? as we have one in the root folder

Reply:
Def will do that in next PR, as it’s used nightly and might need more experiments.

Reply:
i was mainly referring to how the formatting will differ from other crates. We should maintain consistency with other crates

Reply:
Oh, that is mainly the import formats.
But yeah, I agree we should maintain all crates.